crates/wasi-provider/src/lib.rs

```diff
@@ -107,9 +107,9 @@ impl Provider for WasiProvider {
         let handle = std::thread::spawn(move || runtime.run().unwrap());
         let mut handles = self.handles.write().unwrap();
```

Comment:
`let output = runtime.output()`

Reply:
Runtime has been moved into the thread though